psydac/core/bsplines.py

```diff
@@ -176,7 +176,7 @@ def basis_funs_1st_der( knots, degree, x, span ):
     return ders
 
 #==============================================================================
-def basis_funs_all_ders( knots, degree, x, span, n ):
+def basis_funs_all_ders(knots, degree, x, span, n, normalization='B'):
```

Comment:
Please add this line to the docstring, similar to the one of `basis_ders_on_quad_grid`:
```
If called with normalization='M', this uses M-splines instead of B-splines.
```

Reply:
Fixed in [3d32ea6](https://github.com/pyccel/psydac/pull/193/commits/3d32ea6ea706b76e80e6d0f74ca8acf36c47be3b)